<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Minecraft2D</class>
 <widget class="QStackedWidget" name="Minecraft2D">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minecraft2D</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	border-image: url(:/gui/res/gui/default.png);
}
QPushButton:hover{
	border-image: url(:/gui/res/gui/hover.png);
}
QPushButton:pressed{
	border-image: url(:/gui/res/gui/pressed.png);
}</string>
  </property>
  <widget class="QWidget" name="p_main">
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>500</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Minecraft2D.qrc">:/gui/res/gui/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_singleplayer">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>单人游戏</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="p_singleplayer">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="pb_startgame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pb_back">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QListWidget" name="lw_saves"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pb_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pb_new">
      <property name="text">
       <string>新游戏</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="pb_del">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="p_new">
   <widget class="QLineEdit" name="le_savename">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>277</y>
      <width>256</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入存档名称</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_create">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>309</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>创建</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>309</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="p_del">
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>277</y>
      <width>256</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>确定删除?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_yes">
    <property name="geometry">
     <rect>
      <x>372</x>
      <y>309</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>是</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_no">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>309</y>
      <width>128</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>否</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_name"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pb_openfoder">
      <property name="text">
       <string>打开文件夹</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pb_done">
      <property name="text">
       <string>完成</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Minecraft2D.qrc"/>
 </resources>
 <connections/>
</ui>
